rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -62,7 +61,7 @@
     private Map<URI, Transaction> recoveringLRAs = new ConcurrentHashMap<>();
     private Map<URI, ReentrantLock> locks = new ConcurrentHashMap<>();
 
-    private static Map<String, String> participants = new ConcurrentHashMap<>();
+    private Map<String, String> participants = new ConcurrentHashMap<>();
```

Comment:
I guess this is separate to the core JBTM in this PR?

Reply:
Well the bean is ApplicationScoped so using a static makes little difference.

Reply:
Agreed, it looks right not to be static but is it related to https://issues.redhat.com/browse/JBTM-3243 is what I was getting at.

Reply:
Well I think so since the PR is all about getting the semantics of an ApplicationScoped bean correct and using statics in such a bean is counter intuitive.